pytorch_toolkit/nncf/README.md

```diff
@@ -1,73 +1,82 @@
-# Neural Network Compression Framework (NNCF)
+# Neural Network Compression Framework  (NNCF)
```

Comment:
This PR have reverted all changes in .md files from https://github.com/opencv/openvino_training_extensions/pull/211